packages/sitecore-jss-rendering-host/src/devServer.ts

```diff
@@ -196,7 +196,22 @@ export function startDevServer({
     serverOptions.public = tunnelUrl;
   }
 
-  const modulePath = path.join(buildArtifactsPath, serverBundleFileName);
+  type ServerBundleFileName =
```

Comment:
I assume that we don't need to define this type, because `serverBundleFileName` is getting appropriate type automatically